test/test_runner.cc

```diff
@@ -77,6 +77,11 @@ bool isDeathTestChild(int argc, char** argv) {
 
 int TestRunner::runTests(int argc, char** argv) {
   const bool is_death_test_child = isDeathTestChild(argc, argv);
+
+  // Use the recommended, but not default, "threadsafe" style for the Death Tests.
```

Comment:
this comment (ie the original) is not entirely accurate

the threadsafe mode is not default (or recommended) as it makes testing much slower - not sure if we can do anything about that, so perhaps this is a pedantic point - but if there was a way not to do this it would be better